x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_groups.ts

```diff
@@ -75,3 +86,32 @@ export function useFetchSloGroups({
     isRefetching,
   };
 }
+
+const mixKqlWithTags = (kqlQuery: string, tags: SearchState['tags']) => {
+  if (!tags) {
+    return kqlQuery;
+  }
+  const tagsKqlIncluded = tags.included?.join(' or ') || '';
+  const excludedTagsKql = tags.excluded?.join(' or ') || '';
+
+  let tagsQuery = '';
+  if (tagsKqlIncluded && excludedTagsKql) {
+    tagsQuery = `slo.tags: (${excludedTagsKql}) and not slo.tags: (${tagsKqlIncluded})`;
```

Comment:
isn't it the opposite?
```suggestion
    tagsQuery = `slo.tags: (${tagsKqlIncluded}) and not slo.tags: (${excludedTagsKql})`;
```

Reply:
tests proven useful 😄